charts/fybrik/files/adminconfig/infrastructure.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file should be moved to samples/adminconfig directory.

Reply:
Changed